extensions/ql-vscode/src/databases/config/db-config-store.ts

```diff
@@ -56,11 +56,31 @@ export class DbConfigStore extends DisposableObject {
     return this.configPath;
   }
 
+  public async setSelectedDbItem(dbItem: SelectedDbItem): Promise<void> {
+    if (!this.config) {
+      // If the app is trying to set the selected item without a config
+      // being set it means that there is a bug in our code. sp we thow
```

Comment:
Very minor:
```suggestion
      // being set it means that there is a bug in our code, so we throw
```

(something like that ☝🏽 )